style-guide/src/hocs/withLayout.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
This HOC allows us to add the layout and decide on a specific Nav type. It's not used when displaying pages inline in index but is used when they're rendered standalone.